Swat/SwatExternalJavaScriptHtmlHeadEntry.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Stores and outputs an HTML head entry for a JavaScript include
+ *
+ * @package   Swat
+ * @copyright 2006-2016 silverorange
+ * @license   http://www.gnu.org/copyleft/lesser.html LGPL License 2.1
+ */
+class SwatExternalJavaScriptHtmlHeadEntry extends SwatHtmlHeadEntry
+{
+    // {{{ protected function displayInternal()
+
+    protected function displayInternal($uri_prefix = '', $tag = null)
+    {
+        $uri = $this->uri;
+
+        printf(
+            '<script type="text/javascript" src="%s"></script>',
+            $uri
+        );
+    }
+
+    // }}}
+    // {{{ protected function displayInlineInternal()
+
+    protected function displayInlineInternal($path)
+    {
+        echo '<script type="text/javascript">';
+        readfile($path . $this->getUri());
```

Comment:
does this even work with an external resource? Even if it does, it'll be slow because it has to download the file from the server and then out it here.

I think this is just used by the HTML concentrate stuff that combines JS files into one big file. For external resources we shouldn't be doing that anyway. I think it would make sense to just return an empty string here instead and add a comment explaining why.